service/src/builder.rs

```diff
@@ -49,29 +50,51 @@ pub async fn build_crate(
 
     let config = get_config(write)?;
     let manifest_path = project_path.join("Cargo.toml");
-    let mut ws = Workspace::new(&manifest_path, &config)?;
+    let ws = Workspace::new(&manifest_path, &config)?;
+    check_no_panic(&ws)?;
 
-    let current = ws.current_mut().map_err(|_| anyhow!("A Shuttle project cannot have a virtual manifest file - please ensure the `package` table is present in your Cargo.toml file."))?;
+    let mut alpha_packages = Vec::new();
+    let mut next_packages = Vec::new();
 
-    let summary = current.manifest_mut().summary_mut();
-    let is_next = is_next(summary);
+    for member in ws.members() {
+        if is_next(member) {
+            ensure_cdylib(member)?;
+            next_packages.push(member.name().to_string());
+        } else if is_alpha(member) {
+            ensure_binary(member)?;
+            alpha_packages.push(member.name().to_string());
+        }
+    }
 
-    if !is_next {
-        ensure_binary(current.manifest())?;
-    } else {
-        ensure_cdylib(current.manifest_mut())?;
+    let mut runtimes = Vec::new();
+
+    if !alpha_packages.is_empty() {
+        let opts = get_compile_options(&config, alpha_packages, release_mode, false)?;
+        let compilation = compile(&ws, &opts)?;
```

Comment:
Currently this function checks if a crate in the workspace is a shuttle service (next or alpha), and if it is it compiles it and it's dependencies. This PR is a first step that is focused on compiling the workspace, so it simply starts the first service in the vector. That the user may not want to compile and start all the shuttle services in the workspace is a good point, and we should consider how to handle this in the PRs where we enable starting multiple services in local runs and in deployer.

Reply:
In a local run in a workspace you would write `cargo run -p abc` to run the binary for that member. An equivalent way in cargo shuttle would be good, right?

And if running multiple services at once is a goal (as @oddgrd's comment suggests), perhaps some sort of `--all` flag on shuttle run?

I'm not aware of the intended way of structuring a workspace with this feature: Is each workspace member supposed to be its own shuttle project, or will all members be grouped into the same project?

Reply:
>  An equivalent way in cargo shuttle would be good, right?

The longer goal will be for shuttle to start all the services in the correct order. The idea is when all the workspace members relate to a single project - think of a workspace crate `a` calling workspace crate `b`. So for a local run, this will also ensure than an entire project can be developed and tested with a single command

Reply:
@chesedo So crates `a` and `b` would be deployed as two separate shuttle projects (containers)?